api/client/hijack.go

```diff
@@ -196,19 +193,15 @@ func (cli *DockerCli) hijack(method, path string, setRawTerminal bool, in io.Rea
 		defer term.RestoreTerminal(cli.inFd, oldState)
 	}
 
+	receiveStdout := make(chan error)
```

Comment:
I think it should be buffered to avoid goroutine leak, because select at 236 has two ways now.